docs/asciidoc/tcp-client.adoc

```diff
@@ -342,3 +343,70 @@ include::{examplesdir}/uds/Application.java[lines=18..33]
 ----
 <1> Specifies `DomainSocketAddress` that will be used
 ====
+
+== Host Name Resolution
+The `TCP` client uses by default `Netty's` domain name lookup mechanism that resolves a domain name asynchronously,
```

Comment:
Change to (note the period at the end, making this line be its own sentence):

By default, the `TCP` client uses Netty's domain name lookup mechanism, which resolves a domain name asynchronously.